### PR TITLE
fix: changed login to logout on wishlist & cart

### PR DESCRIPTION
**Pull request by masraniJay**
Changed login button to logout on wishlist and cart pages.

### CODE REVIEW

**Review by ShoaibTamboli — APPROVED** [review]
looks good to merge

**Review by shubhmehra — APPROVED** [review]
good to merge

File: cart/cart.html [hunk 1]
```diff
@@ -15,7 +15,7 @@
         <a href="../index.html" class="navigation-logo">Feather-store</a>
         <input class="navigation-search" placeholder="Search">
         <div>
-            <a href="../Authentication/login.html" class="navigation-item">Login</a>
+            <a href="../index.html" class="navigation-item">Logout</a>
             <a href="../wishlist/wishlist.html" class="navigation-item">Whishlist</a>
```
Comment (sashaboi) [inline cart/cart.html:19]: Typo with Wishlist 😊 and create account typo as well hope that helps

**Review by sashaboi — APPROVED** [review]
Looks good to merge  once typos are addressed

**Pull request merged**